app.py

```diff
@@ -112,6 +112,8 @@ def parseGithubRepos(repos):
 @app.route("/widget/<username>")
 def thing(username):
     token = request.args.get('token')
+    primary = request.args.get('primary', 'indigo')
```

Comment:
I'd rather we not clutter things up with extra parameters for theme colors. What do you think @AlexVasiluta @HYPERDRIVE-Motivator @blackreaper02

Reply:
I think that there should be a good default theme and the end user should be able to customize it. I think this would lead nicely to the customization feature we were thinking of doing.

Reply:
yeah but then we need to build the ui. For customization i was mostly thinking about which repos appear and in which order.